### PR TITLE
Jagged Alliance 2: new port

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

**Comment by miqlas** [comment]
Can't get the icons work, so it got a generic icon right now.
interestingly the icon overwrites the .sh file ocntents.... ? O,O!

File: games-strategy/ja2_stracciatella/ja2_stracciatella-0.15.1.recipe [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libSDL$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
```
Comment (korli) [inline games-strategy/ja2_stracciatella/ja2_stracciatella-0.15.1.recipe]: Need cmd:sed

Comment (miqlas) [inline games-strategy/ja2_stracciatella/ja2_stracciatella-0.15.1.recipe]: Ummm... for what?

Comment (miqlas) [inline games-strategy/ja2_stracciatella/ja2_stracciatella-0.15.1.recipe]: ok, i see now.

**Comment by miqlas** [comment]
i forgot the patch, just a second

**Comment by miqlas** [comment]
A Backup Icon, maybe something wrong with the hrdef

**Comment by korli** [comment]
A script is not a binary, you can't add an icon in the resources section, only as attribute.

**Comment by miqlas** [comment]
Oh. Then how can i attach my precious Icon?

**Comment by miqlas** [comment]
Merge?

**Pull request merged**